core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -185,7 +185,7 @@ private function setConstants(): void
         if ('FE' === $this->getMode() && ($session = $this->getSession()) && $this->request->hasPreviousSession()) {
             $session->start();
 
-            \define('BE_USER_LOGGED_IN', $this->tokenChecker->hasBackendUser() && $this->tokenChecker->isPreviewMode());
+            \define('BE_USER_LOGGED_IN', $this->tokenChecker->isPreviewMode());
```

Comment:
I think we should deprecate `BE_USER_LOGGED_IN` (`FE_USER_LOGGED_IN` is already deprecated). And not change this check here as the name BE_USER_LOGGED_IN suggests that there has to be a logged in backend user.

The usages of `BE_USER_LOGGED_IN` in the core have to be changed then to use the token checker.

Reply:
sounds good, do you think it should be part of this PR?

Reply:
see https://github.com/contao/contao/pull/3873